src/H5mpi.c

```diff
@@ -782,4 +782,58 @@ H5_mpio_gatherv_alloc_simple(void *send_buf, int send_count, MPI_Datatype send_t
     FUNC_LEAVE_NOAPI(ret_value)
 } /* end H5_mpio_gatherv_alloc_simple() */
 
+/*-------------------------------------------------------------------------
+ * Function:    H5_mpio_get_file_sync_required
+ *
+ * Purpose:     Retrieve the MPI hint indicating whether the data written
+ *              by the MPI ROMIO driver is immediately visible to all MPI
+ *              ranks.
+ *
+ * Notes:       This routine is designed for supporting UnifyFS that needs
+ *              MPI_File_sync in order to make the written data available
+ *              to all ranks.
+ *
+ * Return:      True when MPI_File_sync is not needed/False otherwise
+ *
+ * Programmer:  Houjun Tang,  April 7, 2022
+ *
+ *-------------------------------------------------------------------------
+ */
+herr_t
+H5_mpio_get_file_sync_required(MPI_File fh, hbool_t *file_sync_required)
+{
+    herr_t   ret_value = SUCCEED;
```

Comment:
Usually the ret_value variable is the last thing to be declared in the variable block